src/components/header/header.js

```diff
@@ -15,22 +21,28 @@ export const Header = ({ name, Logo, MenuItems, ProfileItems }) => {
           <span>{ProfileItems}</span>
         </nav>
       </div>
-      <div className="bottom-header">
-        <div className="app-credits">
-          <span>Un proiect dezvoltat de</span>
-          <a href="https://code4.ro/" target="_blank" rel="noopener noreferrer">
-            <C4RLogoSvg />
-          </a>
-          <span>În parteneriat cu </span>
-          <a
-            href="https://www.gov.ro/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <GovLogoSVG />
-          </a>
+      {bottomHeader ? (
+        <div className="bottom-header">
+          <div className="app-credits">
+            <span>Un proiect dezvoltat de</span>
+            <a
+              href="https://code4.ro/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <Code4Svg />
+            </a>
+            <span>În parteneriat cu </span>
+            <a
+              href="https://www.gov.ro/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <GovroSvg />
+            </a>
+          </div>
         </div>
```

Comment:
Hey @acis There is a separate issue for this part. We already have a PR for it: https://github.com/code4romania/taskforce-fe-components/pull/33